spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
```

Comment:
So I would not test this because this is not your code. Any code coming from a third party library should not be tested by you. You should trust the code you are importing and if not, then you should not use that gem.